### ISSUE TITLE
Add page redirection for email verification

### ISSUE THREAD

**Issue by colomet** [restrictcontentpro/restrict-content-pro#1414]
Each time one of my testing people activate the account, because the email verification open the edit the profile page, they believe he have to change the password again as is empty the space and is the first section they see.

Maybe to be able to choose the page that is open after the verification would create a better path for the users.

**Comment by chrisvanpatten** [comment]
I'd be happy to file a pull request for this (seems like an easy/quick win!)

Quick questions:
+ Would a page dropdown in the admin settings be preferred? Or would a filter be sufficient?
+ Should the query arg (`rcp-message=email-verified`) always be appended? In other words, should the query arg be appended before or after the filter is applied?

And a quick reminder for my future self: [the redirect is here](https://github.com/restrictcontentpro/restrict-content-pro/blob/7632682c7813e774aafbc847890b849e81b86809/includes/member-functions.php#L1398-L1403).

**Comment by chrisvanpatten** [comment]
Also worth noting that the query arg triggers a front-end notice, which is set [here](https://github.com/restrictcontentpro/restrict-content-pro/blob/cf3cc8cfa36ce5297cd97e13c138cf7f18fa9604/includes/template-functions.php#L175-L219).

It looks like this isn't triggered on every front-end page, so if you were to change to a different page you wouldn't have the notice displayed. For that reason I think a filter is probably best, since it might be confusing to non-technical users why the successful verification notice isn't displayed on their custom page if they select from an admin dropdown.

**Comment by colomet** [comment]
Sorry, too complicate cuestions for me

**Comment by mindctrl** [comment]
It opens the page you have set up as Account page. By default that's the page with the `[subscription_details]` shortcode, which doesn't show the password update form. 

I imagine a lot of people go ahead and add the `[rcp_profile_editor]` or `[password_form]` shortcodes to that page to make a single page to manage/view account details.

An easy win, without adding a new setting or filter, could be to update those templates to not show the update fields if `$_GET['email-verified']` is set.

**Comment by chrisvanpatten** [comment]
@mindctrl I can imagine a case where an advanced user would want to redirect to a specific page. For example we recently did a project where, after completing signup, users were prompted to make a donation to charity. Had we enabled email verification, it might have been nice to have that prompt come after the user verified their account.

Another case might be a site that wants to offer users a small gift (a free download, perhaps) after they've successfully verified their account.
